core/domain/topic_domain_test.py

```diff
@@ -857,86 +745,88 @@ def test_update_thumbnail_filename(self):
         self.assertEqual(self.topic.thumbnail_filename, 'img.svg')
         self.assertEqual(self.topic.thumbnail_size_in_bytes, len(raw_image))
 
-    def test_update_thumbnail_bg_color(self):
+    def test_update_thumbnail_bg_color(self) -> None:
         self.assertEqual(self.topic.thumbnail_bg_color, None)
         self.topic.update_thumbnail_bg_color('#C6DCDA')
         self.assertEqual(self.topic.thumbnail_bg_color, '#C6DCDA')
 
     def test_cannot_add_uncategorized_skill_with_existing_uncategorized_skill(
-            self):
+            self) -> None:
```

Comment:
```suggestion
        self
    ) -> None:
```

Reply:
@SD-13 You missed this

Reply:
yep, done

Reply:
sorry, previously I missed this one but I added it this time.